perception/traffic_light_fine_detector/test/test_nodelet.cpp

```diff
@@ -0,0 +1,108 @@
+// Copyright 2024 TIER IV, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include "traffic_light_fine_detector/nodelet.hpp"
+
+#include <gtest/gtest.h>
+
+sensor_msgs::msg::RegionOfInterest createMapBasedBbox(
+  uint32_t x_offset, uint32_t y_offset, uint32_t width, uint32_t height)
```

Comment:
Would you add const for the arguments?
```suggestion
  const uint32_t x_offset, const uint32_t y_offset, const uint32_t width, const uint32_t height)
```

Reply:
Thank you for your comment!
I modified in https://github.com/autowarefoundation/autoware.universe/pull/7075/commits/441a6ff41449bf8b12f805267e2cd1004429b0f6